src/createServer.js

```diff
@@ -1,9 +1,41 @@
 /* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+    const parts = normalizedUrl.pathname.slice(1).split('/').join(', ');
```

Comment:
parts should be an array so remove join method
![image](https://github.com/user-attachments/assets/52dae50a-43ba-4549-ad65-4090f914143f)